src/args.c

```diff
@@ -335,33 +428,79 @@ void parse_args(int argc, char* argv[])
 		// List of implemented arguments
 		if(strcmp(argv[i], "-h") == 0 || strcmp(argv[i], "help") == 0 || strcmp(argv[i], "--help") == 0)
 		{
-			printf("pihole-FTL - The Pi-hole FTL engine\n\n");
-			printf("Usage:    sudo service pihole-FTL <action>\n");
-			printf("where '<action>' is one of start / stop / restart\n\n");
-			printf("Available arguments:\n");
-			printf("\t    debug           More verbose logging,\n");
+			const char *bold = cli_bold();
+			const char *normal = cli_normal();
+			const char *blue = cli_color(COL_BLUE);
+			const char *cyan = cli_color(COL_CYAN);
+			const char *gray = cli_color(COL_GRAY);
+			const char *green = cli_color(COL_GREEN);
+			const char *yellow = cli_color(COL_YELLOW);
+			const char *purple = cli_color(COL_PURPLE);
+
+			printf("%sThe Pi-hole FTL engine - %s%s\n\n", bold, get_FTL_version(), normal);
+			printf("Typically, pihole-FTL runs as a system service and is controlled\n");
+			printf("by %ssudo service pihole-FTL %s<action>%s where %s<action>%s is one out\n", green, purple, normal, purple, normal);
```

Comment:
Should we go for `systemd`'s `systemctl` here? All supported distros run `systemd` by default.

Reply:
`service` is a high-level wrapper script and common to all Linux'es. It wraps `/etc/init.d` scripts, `Upstart`'s `initctl` command, and `systemctl` as well. In several ways its similar to `apt` which is also just a wrapper script that will direct commands to the appropriate tool, automatically elevating itself with `sudo` if need-be. I think using `service` is appropriate here.